app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -893,6 +894,8 @@ protected void onNewIntent(Intent intent) {
                 // photo was deleted and camera is launched again to capture a new photo
                 dispatchTakePictureIntent();
             }
+        } else if (action.equals(ACTION_MESSAGE_PUSH_NOTIFICATION)) {
```

Comment:
I am aware this thing is not required, but lets have.
It would help the developers to use the intent if they want to!